Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -1,8 +1,8 @@
 ﻿@page
 @model Covinator.Pages.PfizerVaccination
 @{
-    ViewData["Title"] = "Home page";
-    var pfizerVaccineDistributionAllocationss = (QuickType.PfizerVaccineDistributionAllocations[])ViewData["PfizerVaccineDistributionAllocations"];
+    ViewData["Title"] = "Pfizer Vaccination";
```

Comment:
Added page title which is displayed on the tab.

Reply:
Source:-
https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-5.0&tabs=visual-studio

Reply:
That title is more descriptive.  Good.